rust_src/src/windows.rs

```diff
@@ -877,4 +882,19 @@ pub fn set_window_start(window: LispObject, pos: LispObject, noforce: LispObject
     pos
 }
 
+/// Return the topmost child window of window WINDOW.
+/// WINDOW must be a valid window and defaults to the selected one.
+/// Return nil if WINDOW is a live window (live windows have no children).
+/// Return nil if WINDOW is an internal window whose children form a
+/// horizontal combination.
+#[lisp_fn(min = "0")]
+pub fn window_top_child(window: LispObject) -> LispObject {
+    let window = window_valid_or_selected(window);
+    if window.is_vertical_combination() {
+        window.contents
+    } else {
+        Qnil
```

Comment:
Do we still need `LispObject::constant_nil()` and `LispObject::constant_t()` ?

Reply:
Good question. I suspect Qt and Qnil are sufficient now.